pset-iterators.js

```diff
@@ -170,6 +207,20 @@ called `shortTermPresidents`, declared with const
 console.log('Problem 6:')
 
 // Add your code below this line
+const shortTermPresidents = []
+
+presidents.forEach(function(element){
+  let startDt = new Date (element.took_office)
+  let endDt = new Date (element.left_office)
+
+  let days = (endDt - startDt)/(1000 * 3600 * 24)
```

Comment:
@rvaswani85 nice approach, it results in a more precise calculation. An alternative approach would be to just compare the 4 digit year, that approach [was used here](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-iterators.js#L204)